nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs

```diff
@@ -1135,15 +1131,15 @@ private void ResponseMimisbrunnr(ActionBase.ActionEvaluation<MimisbrunnrBattle>
                                 .DoOnError(e => Debug.LogException(e));
                         });
 
-                var simulator = new StageSimulator(
+                var simulator = new StageSimulatorV1(
```

Comment:
It should use `StageSimulator` instead of V1 version.

Reply:
OK I will change it! thx 🥰